bootstrap/README.md

```diff
@@ -1,31 +1,31 @@
 # BOOTSTRAP SERVICE
```

Comment:
Since bootstrap service will be in MG, we can revert the words to Magistrala in valid places